### DISCUSSION TITLE
Is `jaxb.properties` actually being read by JAXB 4 implementing Moxy 4.x?

### DISCUSSION THREAD

**Discussion by wilx** [eclipse-ee4j/eclipselink#2298] [Q&A] [answered]
I was looking into the code of JAXB 2.x implementation and I can see where they read `jaxb.properties`. But I cannot find such code reading `jaxb.properties` in Moxy 4.x. Is this functionality gone or implemented elsewhere where I did not look?

If it has been removed, the documentation still seems to refer to it. Maybe the documentation needs updating?

**Comment by wilx** [comment, marked as answer]
It has been removed in 4.0.2 of the API: https://github.com/jakartaee/jaxb-api/commit/2fc2446980711533cf11c31ca535f3f677c389b6

**Comment by rfelcman** [comment]
I think it's about #1309 (_jaxb.properties_ removal).

**Comment by rfelcman** [comment]
Thank You. New documentation issue #2299 created.
